README.md

```diff
@@ -166,11 +170,19 @@ of the notebooks.
 1. If a python execution fails to import a module, make sure to add current
 directory to your PYTHONPATH.
 
-1. If starting the server gives a CONNECTION\_ERROR, make sure MongoDB is
+2. If starting the server gives a CONNECTION\_ERROR, make sure MongoDB is
```

Comment:
That's neat, thank you for the tip! I will go learn more about the Markdown syntax when I get time :)

> 0. Good
> 3. night
> 1. !